search for metrics directly in metrictank, bypassing ES

woodsaj opened the pull request:
closes #33 

Instead of searching for series in ES, search directly in metrictank using metrictank's own index.
This ensures that graphite will continue to work regardless of MetricIndex type metrictank is using.

woodsaj committed:
allow passing build targets to build_all

woodsaj committed:
include netcat in docker image

woodsaj committed:
search for series in metrictank. closes #33

- instead of searching in ES, search for series in metrictanks
internal index store.

woodsaj committed:
update default config

Dieterbe commented:
is it normal that  	462de15 also has changes into the merging of series, and how consolidateBy() is set on targets?

woodsaj commented:
merging is performed by metrictank. and the consolidateBy is still added for all targets.

woodsaj committed:
add hack to support legacy litmus series names still used.

Old versions of the worldping-app still use litmus.* series names.
We need to still suppor these. In the near future we can remove this
hack
